contrib/completion/labgrid-client.bash

```diff
@@ -0,0 +1,35 @@
+# complete labgrid-client
+_labgrid-client()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    local prev=${COMP_WORDS[COMP_CWORD-1]}
+
+    case "$prev" in
+	-p)
+	    COMPREPLY=( $( compgen -W "$(labgrid-client complete places)" -- $cur ) )
```

Comment:
maybe cache the list in a global variable and only rerun the command if COMPREPLY is empty?